asset/css/markbind.css

```diff
@@ -24,6 +24,26 @@ pre > code.hljs {
     border-radius: 5px;
 }
 
+.code-block {
+    position: relative;
+}
+
+.code-block > .code-block-lang {
+    position: absolute;
+    top: 0.4em;
+    right: 0.3em;
+    display: inline-block;
+    padding: 0.25em 0.4em;
+    font-size: 75%;
+    line-height: 1;
+    text-align: center;
+    white-space: nowrap;
+    vertical-align: baseline;
+    border-radius: 0.25rem;
+    color: #f8f8ff;
+    background-color: #b4b4b9;
```

Comment:
Attributes should be sorted in alphabetical order.

Reply:
Noted.